applications/plugins/SofaOpenCL/CMakeLists.txt

```diff
@@ -103,7 +105,7 @@ add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES} ${OTHER_FILES
 add_definitions("-DSOFA_SRC_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/../../..\"")
 
 add_definitions("-DSOFA_BUILD_GPU_OPENCL")
-target_link_libraries(${PROJECT_NAME} ${OPENCL_LIBRARIES} SofaHelper SofaEngine SofaUserInteraction csparse) # taucs taucs_mt system-taucs)
+target_link_libraries(${PROJECT_NAME} ${OPENCL_LIBRARIES} SofaHelper SofaEngine SofaUserInteraction csparse SofaGeneralObjectInteraction SofaGeneralDeformable) # taucs taucs_mt system-taucs)
```

Comment:
And not SofaUserInteraction ?

Reply:
What ?

Reply:
My bad, didn't see SofaUserInteraction was already linked 👌